test/e2e/testdata/ext-auth-http-securitypolicy.yaml

```diff
@@ -47,8 +47,8 @@ spec:
     name: http-with-ext-auth
   extAuth:
     http:
-      backendRefs:
-      - name: http-ext-auth
+      backendRef:
```

Comment:
is there an existing test for `backendRefs` as well ?

Reply:
I want to make sure both `backendRefs` and `backendRef` work for ext auth.